This is to request the publication of promcode-shapes and promcode-vocab files of machine readable form.

tomkamimura opened the pull request:
This is a request the publication of promcode-shapes and promcode-voab files in ttl, nt, jsonld and rdf formats.. A folder that contains these files has been created under ns folder. A couple of _index.md files have been modified to add information on promcode.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/oslc-op/website?pullRequest=305) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/oslc-op/website?pullRequest=305) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/oslc-op/website?pullRequest=305) it.</sub>

berezovskyi commented on content/ns/promcode/promcode-vocab.ttl:
```diff
@@ -0,0 +1,397 @@
+	vann:preferredNamespacePrefix "oslc_promcode" ;
+	dcterms:publisher <https://open-services.net/about/> ;
+	dcterms:source <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/promcode-vocab.ttl> ;
+	dcterms:hasVersion "CSD01" ;
```
The MD file edit says CS02 but this file is at the CSD01 stage. Which is it?

tomkamimura replied:
The correct stage is CS02. I am sorry that the old stage name was in the ttl files. I will close this PR and will create a new one with corrected files.

berezovskyi commented:
@tomkamimura, thank you for the pull request. Please see my question before we can proceed to merge. Also, can I kindly ask you to make a PR to https://github.com/oslc-op/website-content-negotiation/blob/master/config/conneg.yml to enable the content negotiation over HTTP, please?

tomkamimura commented:
The stage names of machine readable files need to be  corrected from CSD01 to CS02. So, I will close this PR and create a new one with the correction.

tomkamimura commented:
Andrew,

I have created a PR to modify conneg.yml per your instruction.

On the stage of documents, the correct stage is CS02. It was our oversight that turtle files did not reflect the latest situation. I modified the stage information of all machine readable files. I have updated all those files. I closed PR #305 and created a new PR #306 to merge the updated files into your master repository.

Regards,

Tom

> 2022/01/20 23:52、Andrew Berezovskyi ***@***.***>のメール:
> 
> 
> @tomkamimura <https://github.com/tomkamimura>, thank you for the pull request. Please see my question before we can proceed to merge. Also, can I kindly ask you to make a PR to https://github.com/oslc-op/website-content-negotiation/blob/master/config/conneg.yml <https://github.com/oslc-op/website-content-negotiation/blob/master/config/conneg.yml> to enable the content negotiation over HTTP, please?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/oslc-op/website/pull/305#issuecomment-1017588376>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AXJDHIRCGUMXO2V6OZX7CJTUXAOR5ANCNFSM5MD57AXA>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> You are receiving this because you were mentioned.
>
